lib/features/stacking/index.js

```diff
@@ -21,29 +21,20 @@ export const position = {
         ) || []
     ).join('-'),
 
-  getBorderRadii: ({ top, bottom, square }, radius) => (
-    top
-      ? {
-        borderBottomLeftRadius: '0px',
-        borderBottomRightRadius: '0px',
-        borderTopLeftRadius: radius,
-        borderTopRightRadius: radius
-      }
-      : bottom
-        ? {
-          borderBottomLeftRadius: radius,
-          borderBottomRightRadius: radius,
-          borderTopLeftRadius: '0px',
-          borderTopRightRadius: '0px'
-        }
-        : square
-          ? {
-            borderRadius: '0px'
-          }
-          : {
-            borderRadius: radius
-          }
-  ),
+  getBorderRadii: ({ top, bottom, left, right, square }, radius) => {
```

Comment:
lmk if you come up with a better/cleaner/more understandable way to deduce what border radii to return, I did a few iterations to arrive at this rule-ish approach.